docs/layouts.rst

```diff
@@ -301,6 +301,21 @@ These ones live under module ``crispy_forms.bootstrap``.
 .. image:: images/field_disabled.png
    :align: center
 
+.. _`input_size`:
+
+Input group size
+----------------
+
+**Input group size**: Adjust the size of an input group (``AppendedText``, ``PrependedText``, ``PrependedAppendedText``) with the appropriate CSS class::
+
+    # Bootstrap 3 - Inputs and spans need size class. Use `css_class`.
+    PrependedText('field_name', StrictButton("Go!"), css_class="input-sm")
+    PrependedText('field_name', StrictButton("Go!"), css_class="input-lg")
+
+    # Bootstrap 4 - Wrapping div needs size class. Use `input_size`.
```

Comment:
If we don't pass `input_size` what happens? I thought that it all still works with the default Bootstrap size, right?

Perhaps this could be added here stating that `input_size` is optional, but that if it's passed then it's used in certain way?

Reply:
@cpina -- you are right but I'm struggling to find the right way to say this succinctly. Can you have a look at my next itteration here?

Reply:
I like it - thanks!